src/AssetManager.php

```diff
@@ -444,6 +444,7 @@ private function registerAssetBundle(string $name, int $position = null): void
                 $this->registerAssetBundle($dep, $pos);
             }
 
+            unset($this->registeredBundles[$name]);
```

Comment:
Is this variable used by reference somewhere?

Reply:
No. This makes the correct order in the array of registered assets. So that the dependent package is located after the dependent ones. For export, this is very important.